packages/delivery-fetch/delivery.d.ts

```diff
@@ -0,0 +1,6 @@
+import type { Client } from '@bugsnag/core'
+import type { Delivery } from '@bugsnag/core/client'
+
+declare const delivery: (client: Client) => Delivery
```

Comment:
Should this include the `fetch` parameter like we do for other request implementations?

e.g. `delivery-xml-http-request`

https://github.com/bugsnag/bugsnag-js/blob/b7d88e1cd09b2941ad3286eede339e7b95714bb3/packages/delivery-xml-http-request/delivery.d.ts#L4

I can see the argument either way — we don't expect users to pass this, so it's probably unnecessary but it does exist 🤷

Reply:
good spot! it definitely should match the implementation